docs/reference/filebeat/filebeat-input-container.md

```diff
@@ -28,10 +34,7 @@ filebeat.inputs:
 1. all [`filestream`](/reference/filebeat/filebeat-input-filestream.md) inputs require a [`unique ID`](/reference/filebeat/filebeat-input-filestream.md#filebeat-input-filestream-id).
 2. container logs use symlinks, so they need to be [`enabled`](/reference/filebeat/filebeat-input-filestream.md#filebeat-input-filestream-prospector-scanner-symlinks).
 3. `paths` is required.
-
-
-
-Due to deprecation of this input, it’s possible to use it only in combination with the `allow_deprecated_use: true` setting as a part of the input configuration.
+::::
```

Comment:
Is this supposed to be here?
```suggestion
```

Reply:
Yes, this block is large, see the preview here https://docs-v3-preview.elastic.dev/elastic/beats/pull/43641/reference/filebeat/filebeat-input-container